<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APRSSettingsDialog</class>
 <widget class="QDialog" name="APRSSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>275</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>APRS Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="igateCallsignLabel">
        <property name="text">
         <string>IGate Callsign</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="altitudeUnitsLabel">
        <property name="text">
         <string>Altitude Units</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="altitudeUnits">
        <property name="toolTip">
         <string>Units used for displaying altitudes</string>
        </property>
        <item>
         <property name="text">
          <string>Feet</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Metres</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="speedUnitsLabel">
        <property name="text">
         <string>Speed Units</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="igateFilterLabel">
        <property name="text">
         <string>IGate Filter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="igateServer">
        <property name="toolTip">
         <string>Name of APRS-IS server to connect to</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>noam.aprs2.net</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>soam.aprs2.net</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>euro.aprs2.net</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>asia.aprs2.net</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>aunz.aprs2.net</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="igateFilter">
        <property name="toolTip">
         <string>Server side filter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="igateSeverLabel">
        <property name="text">
         <string>IGate Server</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="igateCallsign">
        <property name="toolTip">
         <string>Callsign to use when connecting to APRS-IS server</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="igatePasscode">
        <property name="toolTip">
         <string>Passcode corresponding to APRS-IS callsign</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="speedUnits">
        <property name="toolTip">
         <string>Units used for displaying speeds</string>
        </property>
        <item>
         <property name="text">
          <string>Knots</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MPH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KPH</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="igatePasscodeLabel">
        <property name="text">
         <string>IGate Passcode</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="temperatureUnits">
        <property name="toolTip">
         <string>Units used for displaying temperatures</string>
        </property>
        <item>
         <property name="text">
          <string>Fahrenheit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Celsius</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="temperatureUnitsLabel">
        <property name="text">
         <string>Temperature Units</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="rainfallUnitsLabel">
        <property name="text">
         <string>Rainfall Units</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="rainfallUnits">
        <property name="toolTip">
         <string>Units used for displaying rainfall</string>
        </property>
        <item>
         <property name="text">
          <string>Hundredths of an inch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Millimetres</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>APRSSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>APRSSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
